src/main/java/org/jboss/xavier/analytics/pojo/output/workload/summary/WorkloadSummaryReportModel.java

```diff
@@ -66,6 +66,10 @@
     @OneToMany(mappedBy = "report", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
     private Set<WorkloadsDetectedOSTypeModel> workloadsDetectedOSTypeModels;
 
+    @JsonIgnore
```

Comment:
Remove the `JsonIgnore` annotation please otherwise the `scanRunModels` will be ignored when the JSON response is created and so the UI won't have the data.
```suggestion
```